geotrek/signage/templates/signage/signage_detail_attributes.html

```diff
@@ -105,7 +105,7 @@ <h3>{% trans "Blades" %}</h3>
         </table>
         {% if view|class_name == "SignageDetail" %}
             <p style="text-align: right">
-                {% if perms.signage.add_blade %}
+                {% if perms.signage.add_blade and user.profile.structure == object.structure or user.is_superuser or  perms.authent.can_bypass_structure %}
```

Comment:
Un utilisateur peut avoir le droit perms.authent.can_bypass_structure sans avoir le droit perms.signage.add_blade et donc ce cas il ne doit pas pouvoir ajouter de lame. D'autre part, si on est superuser on a automatiquement les autres droits donc je suis pas sûr qu'il soit nécessaire de tester is_superuser. Du coup ça serait plutôt `perms.signage.add_blade and (user.profile.structure == object.structure or  perms.authent.can_bypass_structure)`